### PR TITLE
[algoliasearch/lite] Update getObject types

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/codesandbox/codesandbox-client/blob/72dbe969bfa4ccecced9577843cc46bc5be7253d/packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/VersionEntry/index.tsx#L61-L72
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

CC @nunomaduro

### CODE REVIEW

**Review by nunomaduro — APPROVED** [review]
Thanks!

File: types/algoliasearch/lite/index.d.ts [hunk 1]
```diff
@@ -101,6 +101,11 @@ declare namespace algoliasearch {
       attributes: string[],
       cb: (err: Error, res: {}) => void
     ): void;
+    /**
+     * Gets specific attributes from an object
+     * https://github.com/algolia/algoliasearch-client-js#find-by-ids---getobjects
```
Comment (bodinsamuel) [inline types/algoliasearch/lite/index.d.ts]: This is a dead link
I believe something like that would be better:
https://www.algolia.com/doc/api-reference/api-methods/get-objects/#retrieve-only-one-object

Comment (MichaelDeBoey) [reply types/algoliasearch/lite/index.d.ts]: Updated the links 🙂

**Review by nunomaduro — APPROVED** [review]

File: types/algoliasearch/lite/index.d.ts (hunk 1 above)
Comment (bodinsamuel) [reply types/algoliasearch/lite/index.d.ts]: Thanks a lot 🙏 ❤️

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**